pages/cpm.vue

```diff
@@ -69,14 +87,41 @@ export default {
       if (!L.Browser.ie && !L.Browser.opera && !L.Browser.edge) {
         layer.bringToFront()
       }
+
+      this.info.update(layer._leaflet_id)
+      console.log(layer._leaflet_id)
```

Comment:
remove/comment the console log debug statement before merging?

Reply:
Yes, will do it.